ui/raidboss/data/05-shb/trial/hades.txt

```diff
@@ -92,7 +92,7 @@ hideall "Double"
 903.0 "--sync--" sync / 1[56]:[^:]*:Hades:417F:/
 905.0 "Gaol Add"
 906.5 "--sync--" sync / 14:[^:]*:Hades:416C:/
-936.2 "Chorus Of The Lost" sync / 1[56]:[^:]*:416C:Hades:/
+936.2 "Chorus Of The Lost" sync / 1[56]:[^:]*:Hades:416C:/
```

Comment:
Same situation here, I think this should be 14: etc.

Reply:
I think this one seems unlikely, as there is a 14 line at 906.5, and 906.5/936.2 parallels the later 1068.5/1098.2.